pkg/deploy/resizer/resize.go

```diff
@@ -21,15 +20,12 @@ type DeploymentConfigResizer struct {
 // ResizerFor returns the appropriate Resizer client depending on the provided
 // kind of resource (Replication controllers and deploymentConfigs supported)
 func ResizerFor(kind string, osc *client.Client, kc *kclient.Client) (kubectl.Resizer, error) {
-	switch kind {
-	case "ReplicationController":
-		c := kubectl.NewResizerClient(kc)
-		return kubectl.ResizerFor(kind, c)
-	case "DeploymentConfig":
+	if kind == "DeploymentConfig" {
 		c := NewResizerClient(osc, kc)
 		return &DeploymentConfigResizer{c}, nil
 	}
-	return nil, fmt.Errorf("no resizer has been implemented for %q", kind)
+	c := kubectl.NewResizerClient(kc)
+	return kubectl.ResizerFor(kind, c)
```

Comment:
`ResizerFor` needs a `ResizerClient` to work. Or are you talking about the naming of the variable?